spec/classes/datadog_agent_integrations_elasticsearch_spec.rb

```diff
@@ -90,9 +93,12 @@
           }
         end
 
+
+
         it { is_expected.to contain_file(conf_file).with_content(%r{instances:}) }
```

Comment:
Rubocop is very picky so you need to delete these two blank lines for it to pass
```suggestion
        it { is_expected.to contain_file(conf_file).with_content(%r{instances:}) }
```